src/confirm-modal.ts

```diff
@@ -0,0 +1,53 @@
+import { App, Modal, Setting } from "obsidian";
+
+export class ConfirmModal extends Modal {
+	// Message to show in the modal
+	message: string;
+
+	// Callback to run on user choice
+	callback: (choice: boolean) => void;
+
+	// Whether an option was picked
+	picked: boolean;
+
+	constructor(
```

Comment:
Would you mind making the constructor head one line? It's not really big enough to warrant the waste of vertical space imo